core/src/test/java/org/hibernate/ogm/test/cascade/CascadeEntityLoaderTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.test.cascade;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Date;
+
+import org.hibernate.loader.entity.CascadeEntityLoader;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Testing merge and refresh {@link CascadeEntityLoader}.
+ * Those loader coming in action on subsequent Entity load from data store,
+ * when merging detached Entity or refreshing attached Entity,
+ * performing operation on the Entity itself or from Parent Entity enabling cascade.
+ *
```

Comment:
This comment is very unclear. I'm not even sure to understand what you mean. Could you maybe try to make shorter sentences?

Reply:
You're right. I'm going to cut and rephrase it.